apps/dashboard/src/components/billing/utils/hubspot.constants.ts

```diff
@@ -0,0 +1,3 @@
+export const HUBSPOT_FORM_IDS = {
+  UPGRADE_CONTACT_SALES: '297d0b87-4c09-4ccf-a74c-28fd5e3b37f6',
```

Comment:
We need a VITE environment variable on this to prevent Hubspot from working locally or on staging.

Reply:
This is actually not the hubspot id itself, just an id of the form we use to submit. I would keep it as is, as we have a vite env for hubspot already.